### PR TITLE
replace tsx with ts-blank-space

### PR DESCRIPTION
**Pull request by turadg**
incidental

## Description
Remove `tsx` now that we can use `ts-blank-space` throughout. It aligns better with future `.ts` support in Node.

### Security Considerations
none
### Scaling Considerations
none

### Documentation Considerations
fewer cases, fewer deps

### Testing Considerations
CI

### Upgrade Considerations
none

### CODE REVIEW

File: multichain-testing/package.json [hunk 1]
```diff
@@ -40,7 +40,6 @@
     "patch-package": "^8.0.0",
     "starshipjs": "2.4.1",
     "ts-blank-space": "^0.4.4",
-    "tsx": "^4.15.6",
```
Comment (0xpatrickdev) [inline multichain-testing/package.json:41]: There's 3 more `bin/tsx` calls in the `multichain-testing/Makefile` as well:
https://github.com/Agoric/agoric-sdk/blob/a99e78960229f8d8e2f98f1d6a7b2405138d65bb/multichain-testing/Makefile#L79-L80

Comment (turadg) [reply multichain-testing/package.json:41]: whups thanks. I'll make those run with the shebang

File: multichain-testing/Makefile [hunk 2]
```diff
@@ -98,7 +98,7 @@ tail-slog:
 
 .PHONY: wait-for-pods
 wait-for-pods:
-	node_modules/.bin/tsx scripts/pod-readiness.ts
+	scripts/pod-readiness.ts
```
Comment (0xpatrickdev) [inline multichain-testing/Makefile:101]: Please try `make start` locally - it's exiting early for me with:

```
make start
/Users/0xpatrick/repos/agoric-sdk/multichain-testing/scripts/install.sh --config config.yaml --name agoric-multichain-testing --version v0.2.20
Script dir: /Users/0xpatrick/repos/agoric-sdk/multichain-testing/scripts
"starship" already exists with the same configuration, skipping
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "starship" chart repository
Update Complete. ⎈Happy Helming!⎈
NAME           	CHART VERSION	APP VERSION	DESCRIPTION                
starship/devnet	0.2.20       	0.1.1      	A Helm chart for Kubernetes
name: agoric-multichain-testing, args:  --set-file chains[0].scripts.updateConfig.data=/Users/0xpatrick/repos/agoric-sdk/multichain-testing/scripts/update-config.sh, chart: starship/devnet, version: v0.2.20
NAME: agoric-multichain-testing
LAST DEPLOYED: Mon Dec  2 17:37:00 2024
NAMESPACE: default
STATUS: deployed
REVISION: 1
TEST SUITE: None
scripts/pod-readiness.ts
scripts/pod-readiness.ts: line 1: import: command not found
scripts/pod-readiness.ts: line 2: import: command not found
scripts/pod-readiness.ts: line 4: syntax error near unexpected token `('
scripts/pod-readiness.ts: line 4: `const checkPodsReadiness = async (): Promise<boolean> => {'
make: *** [wait-for-pods] Error 2
```

Comment (turadg) [reply multichain-testing/Makefile:101]: ah, because that didn't have a shebang. fixing

**Review by dckc — APPROVED** [review]
very nice.

**Pull request merged**